PWGHF/HFC/Tasks/femtoDreamPairTaskCharmHadrons.cxx

```diff
@@ -100,22 +100,23 @@ struct HfTaskCharmHadronsFemtoDream {
 
   /// Event selection
   struct : ConfigurableGroup {
-    Configurable<int> multMin{"EventSel.multMin", 0, "Minimum Multiplicity (MultNtr)"};
-    Configurable<int> multMax{"EventSel.multMax", 99999, "Maximum Multiplicity (MultNtr)"};
-    Configurable<float> multPercentileMin{"EventSel.multPercentileMin", 0, "Maximum Multiplicity Percentile"};
-    Configurable<float> multPercentileMax{"EventSel.multPercentileMax", 100, "Minimum Multiplicity Percentile"};
-  } EventSel;
+    std::string prefix = "eventSel";
+    Configurable<int> multMin{"multMin", 0, "Minimum Multiplicity (MultNtr)"};
+    Configurable<int> multMax{"multMax", 99999, "Maximum Multiplicity (MultNtr)"};
+    Configurable<float> multPercentileMin{"multPercentileMin", 0, "Maximum Multiplicity Percentile"};
+    Configurable<float> multPercentileMax{"multPercentileMax", 100, "Minimum Multiplicity Percentile"};
+  } eventSel;
 
   /// Particle 1 (track)
-  Configurable<femtodreamparticle::cutContainerType> trk1CutBit{"trk1CutBit", 5542474, "Particle 1 (Track) - Selection bit from cutCulator"};
-  Configurable<int> trk1PDGCode{"trk1PDGCode", 2212, "PDG code of Particle 1 (Track)"};
-  Configurable<float> trk1PIDThres{"trk1PIDThres", 0.75, "Momentum threshold for PID selection for particle 1 (Track)"};
-  Configurable<femtodreamparticle::cutContainerType> trk1TPCBit{"trk1TPCBit", 4, "PID TPC bit from cutCulator for particle 1 (Track)"};
-  Configurable<femtodreamparticle::cutContainerType> trk1TPCTOFBit{"trk1TPCTOFBit", 2, "PID TPCTOF bit from cutCulator for particle 1 (Track)"};
-  Configurable<float> trk1MaxEta{"trk1MaxEta", 10., "Maximum eta of partricle 1 (Track)"};
-  Configurable<float> trk1MaxPt{"trk1MaxPt", 999., "Maximum pT of partricle 1 (Track)"};
-  Configurable<float> trk1MinEta{"trk1MinEta", -10., "Minimum eta of partricle 1 (Track)"};
-  Configurable<float> trk1MinPt{"trk1MinPt", 0., "Minimum pT of partricle 1 (Track)"};
+  Configurable<femtodreamparticle::cutContainerType> cutBitTrack1{"cutBitTrack1", 5542474, "Particle 1 (Track) - Selection bit from cutCulator"};
+  Configurable<int> PDGCodeTrack1{"PDGCodeTrack1", 2212, "PDG code of Particle 1 (Track)"};
+  Configurable<float> PIDThresTrack1{"PIDThresTrack1", 0.75, "Momentum threshold for PID selection for particle 1 (Track)"};
+  Configurable<femtodreamparticle::cutContainerType> TPCBitTrack1{"TPCBitTrack1", 4, "PID TPC bit from cutCulator for particle 1 (Track)"};
+  Configurable<femtodreamparticle::cutContainerType> TPCTOFBitTrack1{"TPCTOFBitTrack1", 2, "PID TPCTOF bit from cutCulator for particle 1 (Track)"};
```

Comment:
Use lowerCamelCase for variables.

Reply:
I found this line while looking naming conventions ```CamelCase Convention

All names in C++ code follow camel case convention. This is the practice of writing compound words so that each word begins with a capital letter. No underscores are allowed. Acronyms are the only exception : they can be capitalized.```

Reply:
Correct. And the next sentence says:

> Variables and functions start with a lowercase letter.

Reply:
Combined with the HF conventions, it gives you `TpcBitTrack1`, `TpcTofBitTrack1`, which is much more readable.

Reply:
Okay I thought this is exception for "acronyms"